### ISSUE TITLE
packaging: Auto-change datadir in ubuntu ppa

### ISSUE THREAD

**Issue by GusevVasily** [bitcoin/bitcoin#13826]
Hello

When running, bitcoind puts default folder: /var/lib/bitcoin
Changes in the config file and manually specify datadir e help. Blocks are downloaded to /var/lib/bitcoin, and then the demon is painted.
There was a bug in version 0.16.2

**Comment by MarcoFalke** [comment]
How did you install bitcoind?

**Comment by GusevVasily** [comment]
@MarcoFalke
apt upgrade -> bitcoind-0.16.2-xeniall is ready
installing...

and reinstal:
apt install bitcoind -> bitcoind-0.16.2-xeniall installing

OS:
Ubunsu Server 16.04.5 x64

**Comment by GusevVasily** [comment]
Hello ?

**Comment by GusevVasily** [comment]
...8012 /usr/bin/bitcoind -daemon -datadir=/var/lib/bitcoin -conf=/*my_patch/

Where did 'datadir' ?

I started daemon:
/usr/bin/bitcoind -daemon conf=/*my_patch/
No datadir..

**Comment by TheBlueMatt** [comment]
These issues should be filed against the packaging repo, see https://github.com/bitcoin-core/packaging/issues/5

**Comment by TheBlueMatt** [comment]
I'm not actually sure what the issue is here? Did the package overwrite some local settings, or are you just expecting it to put the datadir elsewhere? In any case, please re-file the issue on the packaging repo at https://github.com/bitcoin-core/packaging/issues/new isntead.

**Comment by MarcoFalke** [comment]
Indeed, please file at the packaging repo if this is still an issue

**Comment by GusevVasily** [comment]
@MarcoFalke
I don't really understand you. Let me describe the problem in more detail: I had '0.16.1' kernel version. Everything worked like clockwork. The demon I ran the command '/bin/bash/usr / bin/bitcoind -config'. After upgrading to '0.16.2', the daemon hangs. Look 'htop' I see that the daemon is now loaded by the command: '/bin /bash/usr/bin/bitcoind-datadir "/var/lib / bitcoin" -config'. Ie automatically appeared option 'datadir', although I did not write it anywhere. And in the config file I tried to change this option 'datadir', but to no avail.. Rolled back the kernel package to '0.16.1', everything worked again as before, without adding the daemon file option.
